liferay-tango/liferay-portal
# LPS-118107 Replace TrafficSources endpoint with Acquisition Channel

**Description**

As a first step, we are just replacing the old traffic sources endpoint with the new acquisition channel endpoint.

**How to test it**

Add direct traffic source is quite easy, all the views that don't have a referrer are marked as direct traffic.

If you want to test other traffic sources, you should add to the top head the JS:

```
Object.defineProperty(document, 'referrer', {
	configurable: true,
	get: function() {
		return "your_url.com";
	}
});
```

The events sent to AC will have the referral that you add to your top_head.jsp
